services/afp-protonet.service

```diff
@@ -4,6 +4,7 @@ Description=Time Machine AFPD service
 After=init-protonet.service hostname-avahi-protonet.service
 Requires=init-protonet.service hostname-avahi-protonet.service
 ConditionPathExists=/etc/protonet/soul/enabled
+ConditionPathExists=!/etc/protonet/afpd/disabled
```

Comment:
It is not. afpd is simply enabled by default, in absence of the 'off switch'. Using 'enable' method would require force-enabling on all existing deployments.